Signal/src/ViewControllers/ContactViewController.swift

```diff
@@ -288,10 +258,10 @@ class ContactViewController: OWSViewController, CNContactViewControllerDelegate
 
         var lastView: UIView = nameLabel
 
-        if let firstPhoneNumber = contact.phoneNumbers.first {
+        for phoneNumber in systemContactsWithSignalAccountsForContact() {
```

Comment:
Added convenience methods to DRY and improve readability, since we filter contacts/phone numbers in multiple ways in this class.